package.json

```diff
@@ -21,15 +21,15 @@
     "test:node": "node node_test.js"
   },
   "devDependencies": {
-    "browserify": "^11.0.0",
+    "browserify": "^13.1.0",
     "chrome-launch": "^1.1.3",
     "self-signed-https": "^1.0.5",
-    "send": "^0.12.2",
+    "send": "^0.14.1",
     "tape": "^4.1.0",
     "tape-run": "2.1.4"
   },
   "dependencies": {
-    "superagent": "^1.3.0",
+    "superagent": "^2.2.0",
```

Comment:
The bit in the changelog about promises might be worth looking into. Maybe it can still be used w/o requiring promises?